### ISSUE TITLE
Question about this work!

### ISSUE THREAD

**Issue by DLwbm123** [cswin/AWC#1]
I have a question about paper "CFEA: Collaborative Feature Ensembling Adaptation for Domain Adaptation in Unsupervised Optic Disc and Cup Segmentation". The data you used in this paper is from REFUGE challenge. And you utilize the training set (Zeiss Visucam 500) as source domain while the validation set (Canon CR-2) as target domain. The REFUGE challenge setting is training on Zeiss Visucam 500 images and validatiion on Canon CR-2 images, that means there is no domain shift problem considered in this challenge(but the problem exists actually). Thus, if you take the domain shift problem in consideration, why didn't you cmpare your methods with the best methods reported on the challenge leaderboard? I noticed that the results reported is better than your method in the paper(the challenge methods didn't consider the domain shift problem). Thanks for your excellent work and look forward to your reply!
